src/modules/services_manager_service.rb

```diff
@@ -208,6 +214,16 @@ def enabled service
       end
     end
 
+    # Returns whether the given service can be enabled/disabled by the user
+    #
+    # @param String service
```

Comment:
AFAIK the correct format should be something like:

```ruby
# @param [String] service Service name
```

or as I prefer:

```ruby
# @param service [String] Service name
```
The same for the return value:

```ruby
# @return [Boolean] description if you wish
```

You can check [yard](http://yardoc.org/guides/index.html) docs for mor details.